dbms/src/Flash/EstablishCall.cpp

```diff
@@ -191,6 +191,7 @@ void EstablishCallData::tryConnectTunnel()
 
 void EstablishCallData::write(const mpp::MPPDataPacket & packet)
 {
+    GET_METRIC(tiflash_coprocessor_response_bytes, type_mpp_establish_conn).Increment(packet.ByteSizeLong());
```

Comment:
The sync version is not handled?

Reply:
Can we add get metrics here?
https://github.com/pingcap/tiflash/blob/202f1f545bfdb5ad8f1677e2a3adcc174d2f0820/dbms/src/Flash/Mpp/MPPTunnel.cpp#L146-L147

Reply:
> The sync version is not handled?

I will add metric for sync version

Reply:
> Can we add get metrics here?
> 
> https://github.com/pingcap/tiflash/blob/202f1f545bfdb5ad8f1677e2a3adcc174d2f0820/dbms/src/Flash/Mpp/MPPTunnel.cpp#L146-L147

I think this may include local tunnel which should not be recorded in metric.

Reply:
How about add a new metrics for local tunnel?

Reply:
> How about add a new metrics for local tunnel?

get

Reply:
> How about add a new metrics for local tunnel?

done

Reply:
I think as @SeaRise suggested, the metrics can be updated in https://github.com/pingcap/tiflash/blob/202f1f545bfdb5ad8f1677e2a3adcc174d2f0820/dbms/src/Flash/Mpp/MPPTunnel.cpp#L146-L147

Reply:
> I think as @SeaRise suggested, the metrics can be updated in
> 
> https://github.com/pingcap/tiflash/blob/202f1f545bfdb5ad8f1677e2a3adcc174d2f0820/dbms/src/Flash/Mpp/MPPTunnel.cpp#L146-L147

okk

Reply:
> I think as @SeaRise suggested, the metrics can be updated in
> 
> https://github.com/pingcap/tiflash/blob/202f1f545bfdb5ad8f1677e2a3adcc174d2f0820/dbms/src/Flash/Mpp/MPPTunnel.cpp#L146-L147

okk

> Can we add get metrics here?
> 
> https://github.com/pingcap/tiflash/blob/202f1f545bfdb5ad8f1677e2a3adcc174d2f0820/dbms/src/Flash/Mpp/MPPTunnel.cpp#L146-L147

okk